pkg/standalone-utils/test/VaultActions.test.ts

```diff
@@ -1196,6 +1204,234 @@ describe('VaultActions', function () {
     });
   });
 
+  describe('exit pool in recovery mode', () => {
+    const amountInBPT = fp(1);
+
+    context('when caller is not authorized', () => {
+      it('reverts', async () => {
+        await expect(
+          relayer.connect(other).multicall([
+            await encodeExitPool(vault, relayerLibrary, tokens, {
+              poolKind: PoolKind.WEIGHTED,
```

Comment:
We actually don't care about the pool kind in this case right?
How about using an invalid pool kind just to show that it works regardless of the kind?

Reply:
That doesn't seem to work; low level failure, I guess when mapping the enum. It doesn't care what the pool kind is, but it seems to have to be a valid pool kind.

Reply:
then would it make sense to make it random in the test to convey that it's the same? Or do you think it'd be too much?

Reply:
Seems a little much :) Maybe if I just add a comment?